x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/automated_response_actions.cy.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Agent } from '@kbn/fleet-plugin/common';
+import { APP_ENDPOINTS_PATH } from '../../../../../common/constants';
+import { closeAllToasts } from '../../tasks/close_all_toasts';
+import { createAgentPolicyTask, toggleRuleOffAndOn, visitRuleAlerts } from '../../tasks/isolate';
+import { cleanupRule, loadRule } from '../../tasks/api_fixtures';
+import { ENDPOINT_VM_NAME } from '../../tasks/common';
+import { login } from '../../tasks/login';
+import type { IndexedFleetEndpointPolicyResponse } from '../../../../../common/endpoint/data_loaders/index_fleet_endpoint_policy';
+import {
+  getAgentByHostName,
+  getEndpointIntegrationVersion,
+  reassignAgentPolicy,
+} from '../../tasks/fleet';
+import { changeAlertsFilter } from '../../tasks/alerts';
+
+describe('Automated Response Actions', () => {
+  const endpointHostname = Cypress.env(ENDPOINT_VM_NAME);
```

Comment:
I don't advise the use of this globally loaded Endpoint host. I suggest you create a host via the `cy.task('createEndpointHost')` for use by the test suite and then clean up after the test file is done.